src/nbl/CMakeLists.txt

```diff
@@ -314,6 +314,10 @@ set(NABLA_SRCS_COMMON
 	${NBL_UI_SOURCES}
 )
 
+if(MSVC AND NBL_BUILD_TOOLS)
```

Comment:
ignore `NBL_BUILD_TOOLS` its for building exes

Reply:
@devshgraphicsprogramming tag Arek for these. I didn't write these

Reply:
just remove the `AND NBL_BUILD_TOOLS`

Reply:
done